x-pack/plugins/apm/public/components/app/service_groups/service_group_save/save_button.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EuiButton } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useApmRouter } from '../../../../hooks/use_apm_router';
+import { useFetcher } from '../../../../hooks/use_fetcher';
+import { SaveGroupModal } from './save_modal';
+
+export function ServiceGroupSaveButton() {
+  const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const router = useApmRouter();
+  const history = useHistory();
+  const params = router.getParams(
+    '/service-groups',
+    '/services',
+    '/service-map',
+    history.location,
+    true
+  );
```

Comment:
@dgieselaar is `getParams` documented somewhere (I didn't see it in https://github.com/elastic/kibana/blob/main/x-pack/plugins/apm/dev_docs/routing_and_linking.md).

Reason I ask is because it's not immediately intuitive what any of the arguments are for.

Reply:
It's not, I would expect us to use `useApmParams` or `useAnyOfApmParams`.